library/src/interactor_impl.cxx

```diff
@@ -840,8 +901,10 @@ interactor& interactor_impl::initBindings()
   this->addBinding({mod_t::ANY, "7"}, "set_camera top", "Camera", std::bind(docStr, "Top View camera"));
   this->addBinding({mod_t::ANY, "9"}, "set_camera isometric", "Camera", std::bind(docStr, "Isometric View camera"));
   this->addBinding({mod_t::NONE, "H"}, "toggle ui.cheatsheet", "Others", std::bind(docStr, "Toggle cheatsheet display"));
+#if F3D_MODULE_UI
+  this->addBinding({mod_t::NONE, "Escape"}, "toggle_console", "Others", std::bind(docStr, "Display/Hide console"));
```

Comment:
No command only binding ?

Reply:
Wdym? There is a `toggle_console` command